src/crimson/common/interruptible_future.h

```diff
@@ -120,6 +121,14 @@ auto call_with_interruption_impl(
   if (interrupt_condition) {
     auto [interrupt, fut] = interrupt_condition->template may_interrupt<
       typename futurator_t::type>();
+    crimson::get_logger(ceph_subsys_osd).debug(
```

Comment:
Is it too verbose with debug log as it is printing with every interruptible function?

Probably it worths a special macro so that we can disable it after we find it doesn't break.